Cargo.toml

```diff
@@ -38,6 +38,7 @@ serde = { version = "1.0", optional = true, features = ['derive'] }
 cpp_demangle = { default-features = false, version = "0.4.0", optional = true, features = [
   "alloc",
 ] }
+windows-targets = "0.52.6"
```

Comment:
shouldn't this be gated behind `cfg(windows)`?

Reply:
Done!